README.md

```diff
@@ -40,7 +40,7 @@ This package provides OpenTelemetry-compliant tracing to Javascript applications
 To install Cisco OpenTelemetry Distribution simply run:
 
 ```sh
-npm install cisco-telescope
+npm install @cisco-telescope/cisco-sdk-node
```

Comment:
did we change the decision to have the same cisco-telescope for all sdk's?

Reply:
Hmm I think this format is going to adjusted to all repos soon